spec/cnb/today_rates_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+
+RSpec.describe CNB::TodayRates do
+  let(:today_rates) { CNB::TodayRates.new(CNB::TODAY_RATES_URL) }
+
+  describe '#exchange_rate_for' do
+    before do
+      stub_request(:get, (CNB::TODAY_RATES_URL + "19.10.2015")).
+        to_return(status: 200, body: load_today_fixture)
+
+      stub_request(:get, (CNB::TODAY_RATES_URL + "20.10.2015")).
+        to_return(status: 200, body: load_today_fixture)
+
+      Timecop.freeze('2015-10-19')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.